wp1/selection/models/petscan.py

```diff
@@ -0,0 +1,54 @@
+import logging
+import urllib
+
+import requests
+import validators
+
+from wp1.constants import WP1_USER_AGENT
+from wp1.exceptions import Wp1FatalSelectionError
+from wp1.selection.abstract_builder import AbstractBuilder
+
+logger = logging.getLogger(__name__)
+
+
+class Builder(AbstractBuilder):
+
+  def build(self, content_type, **params):
+    if content_type != 'text/tab-separated-values':
+      raise Wp1FatalSelectionError('Unrecognized content type')
+    if 'url' not in params:
+      raise Wp1FatalSelectionError('Missing required param: url')
+    if not isinstance(params['url'], str):
+      raise Wp1FatalSelectionError('Param `url` was not str')

```

Comment:
```suggestion
      raise Wp1FatalSelectionError('Param `url` is not str')

```
Looks like you don't use past tense for others